app/uk/gov/hmrc/fhregistrationfrontend/views/businessPartners/v2/business_partners_SoleProprietors_utr.scala.html

```diff
@@ -0,0 +1,83 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.fhregistrationfrontend.forms.navigation.Navigation
+@import uk.gov.hmrc.fhregistrationfrontend.config.AppConfig
+@import uk.gov.hmrc.fhregistrationfrontend.views.ViewHelpers
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.fhregistrationfrontend.views.html.layout
+@import uk.gov.hmrc.fhregistrationfrontend.views.html.helpers
+@import uk.gov.hmrc.fhregistrationfrontend.views.html.helpers.SaveForLater
+@import uk.gov.hmrc.fhregistrationfrontend.views.helpers.BackHelper
+@import uk.gov.hmrc.fhregistrationfrontend.views.businessPartners.v2._
+@import uk.gov.hmrc.fhregistrationfrontend.forms.models.BusinessPartnersEnterUniqueTaxpayerReference
+@import uk.gov.hmrc.fhregistrationfrontend.forms.definitions.BusinessPartnersEnterUtrForm.businessPartnerUtrKey
+
+@this(layout: layout, viewHelpers: ViewHelpers)
+
+@(businessPartnersUtrForm: Form[BusinessPartnersEnterUniqueTaxpayerReference], postAction: Call, partnerName: String)(implicit request: Request[_], messages: Messages, config: AppConfig)
+
+@titlePrefix = @{
+    if (businessPartnersUtrForm.hasErrors) Messages("fh.generic.errorPrefix") + " " else ""
+}
+
+@title = @{
+    Messages("fh.businessPartners.SoleProprietor.utr.title")
+}
+
+@heading = @{
+    Messages("fh.businessPartners.SoleProprietor.utr.heading", partnerName)
+}
+
+@layout(title = titlePrefix + title, backUrl = Some("#")) {
+
+    @Html(s"${BusinessPartnersHeader(heading)}")
```

Comment:
String interpolation not needed - @Html(BusinessPartnersHeader(heading)) should work.

Reply:
Removed string interpolation